### PR TITLE
Mention Python 3 in README.md

### PR DESCRIPTION
**Pull request by markvdw**
Someone asked to emphasise that GPflow requires python 3. I think that's not a bad idea.

### CODE REVIEW

**Comment by awav** [comment]
Looks like we have random test... And somehow it passes.

File: README.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ The full list of [contributors](http://github.com/GPflow/GPflow/graphs/contribut
 
 GPflow implements modern Gaussian process inference for composable kernels and likelihoods. The [online user manual](http://gpflow.readthedocs.io/en/latest/) contains more details. The interface follows on from [GPy](http://github.com/sheffieldml/gpy), for more discussion of the comparison see [this page](http://gpflow.readthedocs.io/en/latest/intro.html#what-s-the-difference-between-gpy-and-gpflow).
 
+GPflow uses [TensorFlow](http://www.tensorflow.org) for runnin computations, which allows fast execution on GPUs, and is implemented in Python 3.
```
Comment (jameshensman) [inline README.md]: runnin'

Comment (hughsalimbeni) [reply README.md]: I think it should say python 3.5 or above. 3.4 doesn't work with the enums cf #566

**Comment by hughsalimbeni** [comment]
NB 3.5 and above only though #566

**Pull request merged**